### PR TITLE
Add Portable Shower bauble

### PR DESCRIPTION
**Pull request by OmdaCZ**
Adds a bauble called "Portable Shower", this bauble infinitely applies warp ward potion effect on the wearer.
Its main purpose is lifting the annoyance from warp effects of players in UV+, who still use magic armour with "Warping". Texture is WIP, only has a placeholder rn. Recipe will be added in NHcoremod.

### CODE REVIEW

File: src/main/java/shukaro/warptheory/items/ItemWarpWardAmulet.java
Comment (OneEyeMaker) [inline src/main/java/shukaro/warptheory/items/ItemWarpWardAmulet.java]: Instead of relying on potion duration, it's better to check if the potion is active on the player:
```suggestion
            if (!player.isPotionActive(PotionWarpWard.instance.id))
```

**Review by OneEyeMaker — APPROVED** [review]

**Comment by Dream-Master** [comment]
i just wait for a better texture before merge

**Pull request merged**
